mongodbforms/documents.py

```diff
@@ -357,7 +358,7 @@ def __new__(cls, name, bases, attrs):
         return new_class
 
 
-class BaseDocumentForm(BaseForm):
+class BaseDocumentForm(BaseModelForm):
```

Comment:
I know – and Devcon had a functional check that broke with part of my changes, but no longer do. What additional things would you like to test?